### PR TITLE
enable plugin for python editor

### PR DESCRIPTION
**Pull request by harshad16**
enable plugin for python editor
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

# Related Issues and Dependencies

Related-to: https://github.com/thoth-station/s2i-generic-data-science-notebook/issues/9

# Description

Based on the information https://elyra.readthedocs.io/en/latest/getting_started/installation.html

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ cython = "==0.29.*"
 dask = "*"
 dill = "==0.3.*"
 distributed = "*"
+elyra-python-editor-extension = "*"
```
Comment (pacospace) [inline Pipfile:20]: We need to run `jupyter lab build`, this plugin extension requires that command to work as it is a server extension, at least until Elyra team migrate to jupyterlab3 compatible installation.

Comment (harshad16) [reply Pipfile:20]: the assemble script executes the `jupyter lab build`, https://github.com/thoth-station/s2i-generic-data-science-notebook/blob/ed7004160d96921ba9d63afef8035fb90676ef9b/.s2i/bin/assemble#L14

however the jupyter lab build seems to fail. Need to work on it.

**Review by pacospace — APPROVED** [review]
🥇 💯 👏🏼

**Pull request merged**